client/src/pages/organizations/Show.js

```diff
@@ -325,13 +331,20 @@ class BaseOrganizationShow extends Page {
 		)
 	}
 
+	orgLabel = (organization) => {
+		const shortName = organization.shortName
+		const label = shortName.replace(/\s/g, '')
+		return `reports_${label}`
+	}
```

Comment:
I think this makes the orgLabel not unique. What if you have 2 organizations with the same shortName and view them after eachother? I think the last pagination will be used when going back (but I didn't try it)?